### PR TITLE
Added Profile to Public/data

### PR DESCRIPTION
**Pull request by AntonyKimani**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/943"></a>

### CODE REVIEW

File: public/data/AntonyKimani.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    "milestones": [
+    {
+        "title": "Open Source Contributer",
+        "date": "January 2022",
+        "color": "grey",
+        "description": "Learning Git and Github , actively contributing to Open Source and Collaborations"
+    }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/AntonyKimani.json:35]:
```suggestion
  "milestones": [
    {
      "title": "Web Development",
      "date": "September 2021",
      "color": "grey",
      "description": "Discovered Web Development and currently learning JavaScript"
    },
    {
      "title": "Open Source Contributer",
      "date": "January 2022",
      "color": "grey",
      "description": "Learning Git and Github , actively contributing to Open Source and Collaborations"
    }
  ]
}

```

Comment (AntonyKimani) [reply public/data/AntonyKimani.json:35]: Hello Krishguptadev , I've seen your changes . What should I do now?

**Review by AntonyKimani — COMMENTED** [review]
Thank you for the review changes.

File: public/data/AntonyKimani.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ANTONY KIMANI",
```
Comment (CodingSpecies) [inline public/data/AntonyKimani.json:2]: Is it needed to capitalise the whole name? I think now, but I am not sure what others would say. @AntonyKimani 
Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

File: public/data/AntonyKimani.json (hunk 2 above)
Comment (AntonyKimani) [reply public/data/AntonyKimani.json]: Thanks @CodingSpecies  For pointing that out .

**Review by CodingSpecies — APPROVED** [review]

**Comment by AntonyKimani** [comment]
@CodingSpecies  how do I close the pull request.

File: public/data/AntonyKimani.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Antony Kimani",
+    "type": "personal",
+    "bio": "Frontend Web Developer | Open Source Enthusiast | Blogger ",
```
Comment (Panquesito7) [inline public/data/AntonyKimani.json:4]: I think this space should be removed 😅 What do you think?
```suggestion
    "bio": "Frontend Web Developer | Open Source Enthusiast | Blogger",
```

File: public/data/AntonyKimani.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      "title": "Web Development",
+      "date": "September 2021",
+      "color": "grey",
+      "description": "Discovered Web Development and currently learning JavaScript"
+    },
+    {
+      "title": "Open Source Contributor",
+      "date": "January 2022",
+      "color": "grey",
+      "description": "Learning Git and Github, actively contributing to Open Source and Collaboration"
```
Comment (Panquesito7) [inline public/data/AntonyKimani.json:29]: I think the `h` should be capitalized here 😅 What do you think?
```suggestion
      "description": "Learning Git and GitHub, actively contributing to Open Source and Collaboration"
```

Comment (RonenTGreat) [reply public/data/AntonyKimani.json:29]: > I think the `h` should be capitalized here 😅 What do you think?

I agree, it's more pleasing to the eyes.

**Comment by CodingSpecies** [comment]
> @CodingSpecies how do I close the pull request.

@AntonyKimani there is no need to close your pull request. Once you amend the given changes above, this pull request will be merged. Thank you.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**